Example/Okta_UITests/Okta_UITests.swift

```diff
@@ -14,73 +14,95 @@ import XCTest
 import OktaAuth
 
 class OktaUITests: XCTestCase {
-    var username = ""
-    var password = ""
+    // Update these values along with your Plist config
+    var username = ProcessInfo.processInfo.environment["USERNAME"]!
+    var password = ProcessInfo.processInfo.environment["PASSWORD"]!
+
+    var testUtils: UITestUtils?
+    let app = XCUIApplication()
 
     override func setUp() {
         super.setUp()
+        testUtils = UITestUtils(app)
 
-        // Update these values along with your Plist config
-        username = "{username}"
-        password = "{password}"
-
-        continueAfterFailure = false
+        continueAfterFailure = true
         XCUIApplication().launch()
     }
 
     override func tearDown() {
         super.tearDown()
     }
 
-    func testAuthorizationCodeFlow() {
-        let app = XCUIApplication()
-        let testUtils = UITestUtils(app)
+    func loginAndWait() {
+        guard let testUtils = testUtils else { return }
+
+        // Check to see if there are tokens displayed (indicating an authenticated state)
+        if let tokens = testUtils.getTextViewValue(label: "tokenView"), tokens.contains("Access Token") {
+            return
+        }
 
         app.buttons["Login"].tap()
 
         // Wait for browser to load
         // This sleep bypasses the need to "click" the consent for Safari
         sleep(2)
+        
+        // Known bug with iOS 11 and system alerts (need to tap twice)
+        app.tap()
+        app.tap()
```

Comment:
This is a **crazy** hack to get around a number of issues with iOS 11 + Swift + `xcodebuild`